components/producers/python-pip-safety/main.go

```diff
@@ -12,13 +12,13 @@ import (
 
 func parseIssues(out []types.Vulnerability) []*v1.Issue {
 	issues := []*v1.Issue{}
+
 	for _, r := range out {
 		issues = append(issues, &v1.Issue{
-			Target:     fmt.Sprintf("%s:%s", r.PackageName, r.AnalyzedVersion),
+			Target:     producers.GetPURLTarget("pypi", "", r.PackageName, r.AnalyzedVersion, nil, ""),
```

Comment:
hint, instead of the string "pypi", you can use [the relevant type from package-url go](https://github.com/package-url/packageurl-go/blob/master/packageurl.go#L54)